test_requirements.txt

```diff
@@ -25,6 +25,7 @@ Pillow>=7.1.2, <7.2
 pre-commit>=1.17.0, <2.0.0
 psutil==5.6.6
 pyarrow>=0.12.0, <1.0.0
+pydantic>=1.5, <1.6.0
```

Comment:
This should also be included in `setup.py` under extras require, so users can do sth like `pip install kedro[validation]` and easily install the dependency they need for the decorator to work. Also worth adding a note about that installation step in the docstrings.

Reply:
thanks, will do